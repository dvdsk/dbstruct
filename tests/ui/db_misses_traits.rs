use dbstruct::dbstruct;

#[dbstruct(db=hashmap)]
struct Test {
    field: Vec<u8>,
}

fn main() {}
