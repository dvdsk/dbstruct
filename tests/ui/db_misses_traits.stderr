error: The database backend (hashmap) you specified can not support all the structs fields

         = help: You need a backend that implements all of these traits: {Ordered}.
       Database backends that implement those traits: sled

 --> tests/ui/db_misses_traits.rs:3:15
  |
3 | #[dbstruct(db=hashmap)]
  |               ^^^^^^^
